docs/cmd/tkn_pipeline_logs.md

```diff
@@ -32,10 +32,11 @@ Show pipeline logs
 ### Options
 
 ```
-  -a, --all      show all logs including init steps injected by tekton
-  -f, --follow   stream live logs
-  -h, --help     help for logs
-  -l, --last     show logs for last run
+  -a, --all         show all logs including init steps injected by tekton
+  -f, --follow      stream live logs
+  -h, --help        help for logs
+  -l, --last        show logs for last run
+  -L, --limit int   lists an amount of pipelineruns (default 5)
```

Comment:
minor: `List number of pipelineruns` instead of `amount`? :)

Reply:
Changed to this in latest push